### ISSUE TITLE
PHP fatal error when running "generate"

### ISSUE THREAD

**Issue by bretrzaun** [dauxio/daux.io#444]
**Describe the bug**
Running 'generate' reports:
```
Fatal error: Declaration of Todaymade\Daux\Console\Generate::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output) must be compatible with Symfony\Component\Console\Command\Command::execute(Symfony\Component\Console\Input\InputInterface $input, Symfony\Component\Console\Output\OutputInterface $output): int in /app/vendor/daux/daux.io/libs/Console/Generate.php on line 53
Script daux generate handling the doc event returned with error code 255
```

**To Reproduce**
1. Create project with dependencies
- symfony/console (v7.1.4)
- daux/daux.io (0.22.2)
- 
2. run 'daux generate'

**Comment by bretrzaun** [comment]
This has been unnoted because according to the composer.json "symfony/console" ^7 is allowed, but composer.lock sticks the version to 6, which is why the tests will work.

**Comment by axelhahn** [comment]
I had the same problem on a fresh installation on debian 11 and php 8.2 when calling `daux`only.

`composer global require daux/daux.io`  installed symfonie v7.1.5.

When I grep a bit then there is the the variable type of the return value is new:
vendor/symfony/console/Command/CompleteCommand.php:71:    protected function execute(InputInterface $input, OutputInterface $output): **int**
vendor/daux/daux.io/libs/Console/Generate.php:53:    protected function execute(InputInterface $input, OutputInterface $output)

The execute() methods need to be updated in 
vendor/daux/daux.io/libs/Console/ClearCache.php:17
vendor/daux/daux.io/libs/Console/Generate.php:53
vendor/daux/daux.io/libs/Console/Serve.php:24

**Comment by axelhahn** [comment]
Named 3 modifications are in the pull request https://github.com/dauxio/daux.io/pull/446

**Comment by onigoetz** [comment]
Fixed by #444
